lib/brain.js

```diff
@@ -1951,9 +1950,7 @@ Brain.prototype.isTierCompliant = function isTierCompliant (tier) {
     case 'custom':
       const customInfoRequestId = tier
       const customerData = _.find(['info_request_id', customInfoRequestId])(customer.customInfoRequestData)
-      if (!customerData) return false
-      if (customerData.approved === true) return true
-      if (customerData.approved === false) return false
+      if (!customerData || customerData.approved === false) return false
```

Comment:
If `customerData` always contains the field `approved`, this can just be replaced to `!!customerData.approved` I think